tests/acceptance/test_update.py

```diff
@@ -267,10 +267,15 @@ def test_too_big_image_update(self, bitbake_variables, connection):
             output = connection.run(
                 "mender %s /var/tmp/image-too-big.mender ; echo 'ret_code=$?'"
                 % install_flag
-            ).stderr
+            )
 
-            assert "no space left on device" in output, output
-            assert "ret_code=0" not in output, output
+            assert any(
+                [
+                    "no space left on device" in out
+                    for out in [output.sterr, output.stout]
```

Comment:
btw, should it not be `stderr` and `stdout` ?

Reply:
:man_facepalming:

Reply:
Pushed a fix. Thanks!